explorer/app/apis/catalog/altos-labs-catalog/common/entities.ts

```diff
@@ -22,18 +22,18 @@ export type AltosLabsCatalogEntity =
   | AltosLabsCatalogFile;
 
 export interface AltosLabsCatalogExperiment extends AltosLabsCatalog {
-  files: string[];
+  s3Uris: string[];
 }
 
 export interface AltosLabsCatalogFile {
   assay: string[];
   description: string;
   doi: string;
   experimentType: EXPERIMENT_TYPE;
-  filePath: string;
   fileType: string;
   initiative: string;
   NCBITaxonomyID?: string[];
+  s3Uri: string;
```

Comment:
These are all abbreviations so we can use S3URI. I guess by that rule doi should be DOI.